### ISSUE TITLE
travis-ci builds failing on cross-gdb

### ISSUE THREAD

**Issue by cpackham** [crosstool-ng/crosstool-ng#754]
When they don't timeout some of the samples build on travis-ci.org fail on the cross-gdb step. This appears to be related to the fact that gdb 8.0 requires C++ support. I can't reproduce the problem locally (kubuntu 16.04) so it may be related to the fact that the travis nodes are running 14.04.

https://travis-ci.org/crosstool-ng/crosstool-ng/jobs/243352859

````
[ERROR]    configure: error: *** A compiler with support for C++11 language features is required.
[ERROR]    make[3]: *** [configure-gdb] Error 1
[ERROR]    make[3]: *** Waiting for unfinished jobs....
[ERROR]    make[2]: *** [all] Error 2
````

**Comment by cpackham** [comment]
More from the build log to save someone else digging

````
[ALL  ]    checking whether x86_64-build_pc-linux-gnu-g++ -O2 -g -pipe -I/home/travis/build/crosstool-ng/crosstool-ng/.build/aarch64-unknown-linux-gnu/buildtools/include -L/home/travis/build/crosstool-ng/crosstool-ng/.build/aarch64-unknown-linux-gnu/buildtools/lib supports C++11 features with -h std=c++11... no
[ERROR]    configure: error: *** A compiler with support for C++11 language features is required.
[ERROR]    make[3]: *** [configure-gdb] Error 1
[ERROR]    make[3]: *** Waiting for unfinished jobs....
````

**Comment by stilor** [comment]
We need to have  g++ installed in the Travis VM, I guess. But I largely gave up on Travis, because with its 50min time limit (and unpredictable time it take to build a given sample, because VMs share disk/memory/CPU load with other VMs) it often causing spurious build failures.

**Comment by cpackham** [comment]
I have another series I'm working on that might sort the 50 minute thing

**Comment by stilor** [comment]
Should be fixed now.

**Comment by cpackham** [comment]
Indeed fixed by #757
